.eslintrc

```
@@ -0,0 +1,7 @@
+{
+  "extends": "airbnb-base",
+  "env": {
+    "browser": true,
+    "es6": true
+  }
```

Could you add the following rules, to ensure some ease in coding style and avoid affecting the already written code?

```suggestion
  },  
  "rules": {
    "comma-dangle": ["error", "always-multiline"],
    "quote-props": ["error", "consistent"],
    "arrow-parens": 0,
    "no-confusing-arrow": 0,
    "no-nested-ternary": 0,
    "implicit-arrow-linebreak": 0,
    "arrow-body-style": 0
  }
```